### PR TITLE
Added Samim

### PR DESCRIPTION
**Pull request by aziz**
Added Persian Samim font

### CODE REVIEW

File: Casks/font-samim.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask 'font-samim' do
+  version '0.10.2'
+
+  sha256 '802d687286d1f47c3797d1c8bd4c7b70fc35735a67e41a99b1a371ecbb0ea5fb'
+  url "https://github.com/rastikerdar/samim-font/releases/download/v#{version}/samim-font-v#{version}.zip"
+
+  homepage 'http://rastikerdar.github.io/samim-font'
+  appcast 'https://github.com/rastikerdar/samim-font/releases.atom',
+    :sha256 => 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
```
Comment (vitorgalvao) [inline Casks/font-samim.rb]: This no longer exists.

File: Casks/font-samim.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  license :ofl
+
+  font "Samim.ttf"
```
Comment (vitorgalvao) [inline Casks/font-samim.rb]: Double quotes are reserved for cases where string interpolation is needed.

File: Casks/font-samim.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'font-samim' do
+  version '0.10.2'
+
```
Comment (vitorgalvao) [inline Casks/font-samim.rb]: Styling is wrong.

File: Casks/font-samim.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/font-samim.rb]: You’re also miscalculating. This value is invalid (it’s the sha256 for an empty string).

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in ed48cbd0d7ed6286417ba27e012929fa8434cdb0. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by vitorgalvao** [comment]
You should always run `brew cask audit {{you_cask}}` and `brew cask style --fix {{your_cask}}` prior to submission. Those’ll catch most mistakes.

**Comment by aziz** [comment]
Thanks for the info. This is my first contribution to Cask and Brew repos. Sorry for being sloppy.

**Comment by vitorgalvao** [comment]
No worries. Thanks again for the contribution.

**Pull request closed without merging**